posts/2019-10-15-Rustup-1.20.0.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Announcing Rustup 1.20.0"
+author: The Rustup Working Group
+---
+
+The rustup working group is happy to announce the release of rustup version 1.20.0. [Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of rustup installed, getting rustup 1.20.0 is as easy as:
+
+```
+rustup self update
+```
+
+If you don't have it already, you can [get rustup][install] from the appropriate page on our website.
+
+[rust]: https://www.rust-lang.org
+[install]: https://rustup.rs
+
+## What's new in rustup 1.20.0
+
+The highlights of this release are profiles support, the ability to get the latest available nightly with all the components you need, and improvements to the `rustup doc` command. You can also check out [the changelog][changelog] for a list of all the changes included in this release.
+
+[changelog]: https://github.com/rust-lang/rustup.rs/blob/master/CHANGELOG.md
+
+### Profiles
+
+Previous versions of rustup installed a few components by default along with each toolchain: the compiler (`rustc`), the package manager (`cargo`), the standard library (`rust-std`), and offline documentation (`rust-docs`). While this approach is fine while developing software locally, some of the components (like `rust-docs`) slowed down the installation due to the large amount of installed files, especially on CIs and on Windows.
```

Comment:
"installation due to the large amount of installed files" I think this is only true on Windows, elsewhere it is just the size which makes things slow. It's unnecessary on CI, but not any slower than it would be locally.

Reply:
It's true on *NIX platforms with latent storage (e.g. HPC clusters with NFS homedirs - and yes, we had someone with that on the issue when we were discussing mitigation strategies)